metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -383,7 +408,7 @@ public void execute(Tuple tuple) {
     }
   }
 
-  protected void handleError(byte[] originalMessage, Tuple tuple, Throwable ex, OutputCollector collector) {
+  protected void handleError(Object originalMessage, Tuple tuple, Throwable ex, OutputCollector collector) {
```

Comment:
Does this need to be relaxed to Object? It looks to me like byte[] is still used everywhere for original message.

Reply:
Sorry, that was from an earlier rev. where I was using that.